flake.nix

```
@@ -0,0 +1,71 @@
+{
+  description = "An example Golang package";
+
+  # Nixpkgs / NixOS version to use.
+  inputs.nixpkgs.url = "nixpkgs/nixos-21.11";
+
+  outputs = { self, nixpkgs }:
+    let
+
+      # to work with older version of flakes
```

Consistently capitalize comments and end with a period:
```suggestion
      # To work with older version of flakes.
```

---

This conversation was resolved without changes because this comment text is auto-generated.